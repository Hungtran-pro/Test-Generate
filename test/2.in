3
34
41
10
