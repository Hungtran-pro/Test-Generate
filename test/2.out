819609360
519664494
275186727
